### PR TITLE
Created reddit search route

### PR DESCRIPTION
**Pull request by ncen5293**
let users search a company by new on reddit

### CODE REVIEW

File: server/routes/reddit.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        console.log(submission.selftext);
+        submissions.push({
+          title: submission.title,
+          platform: submission.subreddit_name_prefixed,
+          link: submission.permalink,
+          image: submission.thumbnail,
+          desc: submission.selftext
+        });
+      });
+      res.send({ success: true, submissions });
```
Comment (derekjou) [inline server/routes/reddit.js:30]: Have you tested this response with postman? I don't know if you can send submissions without a key (e.g. res.send({ success: true, submissions: submissions }); )?

**Review by derekjou — COMMENTED** [review]
I'm assuming the console logs are for debugging, but everything else looks pretty solid for now!

File: server/routes/reddit.js (hunk 1 above)
Comment (ncen5293) [reply server/routes/reddit.js:30]: it works with postman and javascript automatically does that if both of the fields are the same name

**Review by derekjou — APPROVED** [review]
Looks ready to merge!

**Pull request merged**